.ci/code_generation.py

```diff
@@ -65,41 +25,8 @@
             os.remove(f)
     except:
         pass
-core.start_local_server(config=core.AvailableServerConfigs.LegacyGrpcServer)
-code_gen = core.Operator("python_generator")
-code_gen.connect(1, TARGET_PATH)
-for lib in LIB_TO_GENERATE:
-    try:
-        code_gen.connect(0, lib)
-        if lib != LIB_TO_GENERATE[0]:
-            code_gen.connect(2, False)
-        else:
-            code_gen.connect(2, True)
-        print(f"Generating {lib} operators for server {core.SERVER.version}...")
-        code_gen.run()
-        time.sleep(0.1)
-    except Exception as e:
-        print(f"Could not generate operators for library {lib}:\n{str(e)}")
-        raise e
 
-for lib in LIB_OPTIONAL_TO_GENERATE:
-    try:
-        code_gen.connect(0, lib)
-        if lib != LIB_OPTIONAL_TO_GENERATE[0]:
-            code_gen.connect(2, False)
-        else:
-            code_gen.connect(2, True)
-        print(f"Generating optional {lib} operators for server {core.SERVER.version}...")
-        code_gen.run()
-        time.sleep(0.1)
-    except Exception as e:
-        print(f"Could not generate operators for optional library {lib}:\n{str(e)}")
-
-# Reorder imports alphabetically in __init__.py files to reduce changes raised
-for init_file_path in glob.glob(os.path.join(TARGET_PATH, "**/__init__.py"), recursive=True):
-    with open(init_file_path, "r") as init_file:
-        lines = init_file.readlines()
-    with open(init_file_path, "w") as init_file:
-        init_file.writelines(sorted(lines))
+core.set_default_server_context(core.AvailableServerContexts.premium)
+core.start_local_server(config=core.AvailableServerConfigs.LegacyGrpcServer)
```

Comment:
why legacy??

Reply:
@cbellot000 that is historical but we could try with the `GrpcServer`. In a next PR.